main.go

```diff
@@ -147,6 +147,8 @@ func parseFlags(checker *errcheck.Checker, args []string) ([]string, int) {
 	var excludeFile string
 	flags.StringVar(&excludeFile, "exclude", "", "Path to a file containing a list of functions to exclude from checking")
 
+	flags.StringVar(&checker.Mod, "mod", "", "module download mode to use: readonly or vendor.See 'go help modules' for more.")
```

Comment:
Small nit, there is a space missing in the help text.

Reply:
fixed